photo.go

```diff
@@ -65,6 +76,32 @@ func (p *ImgixPhoto) GetThumbnailForWidthAndHeight(w, h int) string {
 	return fullUrl.String()
 }
 
+func (p *ThumborRescaledPhoto) GetPhotoForWidth(w int) string {
+	thumborOptions := gothumbor.ThumborOptions{Width: w, Smart: true}
+	thumborPath, err := gothumbor.GetCryptedThumborPath(p.Secret, p.Key, thumborOptions)
+	if err != nil {
+		fmt.Print(err)
+	}
+
+	parts := []string{p.BaseUrl.String(), thumborPath}
+	thumborUrl := strings.Join(parts, "/")
```

Comment:
Might be better to use `fullUrl := p.BaseUrl.ResolveReference(keyPathUrl).String()` as then we don't have to deal with weird edge cases in joining the Urls. You can see how it's done in `GetPhotoForWidth` for the `ImgixScaledPhoto` type.

Reply:
i was looking for something like this!